lib/cc/analyzer/container.rb

```diff
@@ -116,7 +154,11 @@ def reap_running_container
       end
 
       def timeout
-        (ENV["CONTAINER_TIMEOUT_SECONDS"] || DEFAULT_TIMEOUT).to_i
+        ENV.fetch("CONTAINER_TIMEOUT_SECONDS", DEFAULT_TIMEOUT).to_i
```

Comment:
@GordonDiggs a `0` timeout would result in immediate failure as would `0` max bytes. I think these inputs would result in the same behavior with the `fetch` or the `||`.